src/modules/ekf2/test/test_EKF_height_fusion.cpp

```diff
@@ -182,7 +182,7 @@ TEST_F(EkfHeightFusionTest, gpsRef)
 	EXPECT_NEAR(baro_status.bias, baro_initial + baro_increment, 1.3f);
 
 	const BiasEstimator::status &rng_status = _ekf->getRngHgtBiasEstimatorStatus();
-	EXPECT_NEAR(rng_status.bias, 0.f, 1.f);
+	EXPECT_NEAR(rng_status.bias, 0.f, 1.1f); // TODO: why?
```

Comment:
Most likely a tiny change in convergence speed due to the lower limit on the accel bias change